test/src/com/xilinx/rapidwright/device/TestUnisimPlacements.java

```diff
@@ -32,9 +35,24 @@
 
 public class TestUnisimPlacements {
 
+    private static Set<FamilyType> unsupportedTypes = null;
+
+    static {
+        unsupportedTypes = new HashSet<>();
```

Comment:
What's the significance of these unsupported families? Are they not currently supported by RapidWright at all? Or do they just cause this particular test to fail?

Reply:
Just added a comment to clarify.  Adding support is a bit more involved, will hopefully add in 2023.1.0.